src/components/Settings/SettingsNewAddressbook.vue

```diff
@@ -21,18 +21,15 @@
 -->
 
 <template>
-	<form id="new-addressbook-form" name="new-addressbook-form" class="new-address-book"
-		@submit.prevent="addAddressBook">
-		<input id="new-address-book" ref="addressBook" class="new-address-book-input"
-			placeholder="Address book name"
+	<form id="new-addressbook-form" name="new-addressbook-form" class="new-addressbook"
+		@submit.prevent.stop="addAddressbook">
+		<input id="new-addressbook" ref="addressbook" :pattern="addressBookRegex"
+			class="new-addressbook-input"
+			:placeholder="t('contacts', 'Address book name')"
 			type="text"
 			autocomplete="off" autocorrect="off"
-			spellcheck="false"
-			tooltip-enable="!newAddressBookForm.$pristine"
-			tooltip-trigger="none"
-			tooltip-placement="top"
-			uib-tooltip="Only these special characters are allowed: -_.!?#|()">
-		<input type="submit" value="" class="newAddressBookSubmit inline-button icon-confirm action pull-right">
+			spellcheck="false">
+		<input type="submit" value="" class="newAddressbookSubmit inline-button icon-confirm action pull-right">
```

Comment:
newAddressbookSubmit should probably be written with bem syntax?
Also: `inline-button`, `action` and `pull-right` are not needed :)